client/src/components/ContactCard.vue

```diff
@@ -32,15 +35,41 @@ export default {
 .contact-card {
   header {
     @include flex(row, space-between, center);
-    padding: 0 24px;
+    padding: 0 15px;
+    border-bottom: 2px solid #f4f2f2;
```

Comment:
@hlhartley For all these color hex codes, please add them to the colors.scss file as SASS vars. 

Because of the vue.config.js file, these colors will be available automatically in all components without needing to import the file explicitly.

Reply:
I just added these to the file. Thanks!